target/linux/realtek/dts/rtl9303_xikestor_sks8300-8x.dts

```diff
@@ -432,3 +440,18 @@
 		};
 	};
 };
+
+&thermal_zones {
+	sfp-thermal {
+		polling-delay-passive = <10000>;
+		polling-delay = <10000>;
+		thermal-sensors = <&sfp0 &sfp1 &sfp2 &sfp3 &sfp4 &sfp5 &sfp6 &sfp7>;
```

Comment:
Same as previous commit. Please make this a list `<&sfp0>, ...`

Reply:
will fix. Thanks a lot for the thorough review!